### DISCUSSION TITLE
As a new user of atom, it's not obvious how to share state between React components

### DISCUSSION THREAD

**Discussion by synox** [pmndrs/jotai#2819] [General]
Looking at the start page of https://jotai.org/ – all the code seem like they are in the same file (no exports). Same for the tutorial and quick start. Reading through this for the first time made it not clear what is the advantage over useState from react. 

The page shows the API of the getter and setter, and it mentions "global React state management" but it does not explain how to use it between components and files. 

I would like to see the page updated the explain this better for new users.

**Comment by sandren** [comment]
That's a great point. We'll update the page to better demonstrate how atoms can be used globally. Thanks for the feedback. 🙂
